### PR TITLE
Update new versions of the stacks

### PR DESCRIPTION
**Pull request by acelina**
Adds new versions of NodeJS, NETCORE, PHP, and Ruby which are currently available in Azure config lists

### CODE REVIEW

**Comment by yaohaizh** [comment]
@acelina Thanks for your contribution.  @weidongxu-microsoft could you help take a look at this file to see whether we need update app service spec. Thanks.

File: src/ResourceManagement/AppService/RuntimeStack.cs [hunk 1]
```diff
@@ -7,6 +7,21 @@ namespace Microsoft.Azure.Management.AppService.Fluent
     /// </summary>
     public partial class RuntimeStack
     {
+        public static readonly RuntimeStack NodeJS_LTS = new RuntimeStack("NODE", "LTS");
+        public static readonly RuntimeStack NodeJS_10_LTS = new RuntimeStack("NODE", "10 LTS");
```
Comment (weidongxu-microsoft) [inline src/ResourceManagement/AppService/RuntimeStack.cs]: I am not sure whether Azure support this version "10 LTS" (it is not on portal). Would you refer me to the source of the config lists?

Comment (weidongxu-microsoft) [reply src/ResourceManagement/AppService/RuntimeStack.cs]: Seems should be "10-lts"? (result from "az webapp list-runtimes")

Comment (acelina) [reply src/ResourceManagement/AppService/RuntimeStack.cs]: Thanks for the good catch. I had a wrong reference list. 

Updated the list using ```az webapp list-runtimes --linux```. But now the question is, what about the Windows stack. It gives a different result? Is this class OS dependent?

Comment (weidongxu-microsoft) [reply src/ResourceManagement/AppService/RuntimeStack.cs]: I think on current code design, withBuiltInImage(RuntimeStack) works for Linux only (it continues on flow after withExistingLinuxPlan or withNewLinuxPlan). Windows would initial a different flow, which continues as withPythonVersion etc.

Please give me some more time to confirm it and try out some test cases myself.

Thanks.

File: src/ResourceManagement/AppService/RuntimeStack.cs [hunk 2]
```diff
@@ -15,15 +32,26 @@ public partial class RuntimeStack
...
         public static readonly RuntimeStack PHP_7_0 = new RuntimeStack("PHP", "7.0");
+        public static readonly RuntimeStack PHP_7_2 = new RuntimeStack("PHP", "7.2");
+        public static readonly RuntimeStack PHP_7_3 = new RuntimeStack("PHP", "7.3");
         public static readonly RuntimeStack NETCore_V1_0 = new RuntimeStack("DOTNETCORE", "1.0");
         public static readonly RuntimeStack NETCore_V1_1 = new RuntimeStack("DOTNETCORE", "1.1");
-        public static readonly RuntimeStack Ruby_2_3 = new RuntimeStack("RUBY", "2.3");
+        public static readonly RuntimeStack NETCore_V2_0 = new RuntimeStack("DOTNETCORE", "2.0");
+        public static readonly RuntimeStack NETCore_V2_1 = new RuntimeStack("DOTNETCORE", "2.1");
+        public static readonly RuntimeStack NETCore_V2_2 = new RuntimeStack("DOTNETCORE", "2.2");
+        public static readonly RuntimeStack Ruby_2_3 = new RuntimeStack("RUBY", "2.3.8");
```
Comment (weidongxu-microsoft) [inline src/ResourceManagement/AppService/RuntimeStack.cs]: We would use major version here. So for RUBY we will have 2.3, 2.4, 2.5, 2.6.
The reason is that minor version might change too quick for us to update in code.

With major version, server would figure out the mirror version.

Comment (acelina) [reply src/ResourceManagement/AppService/RuntimeStack.cs]: Fixed. Thanks!

**Comment by weidongxu-microsoft** [comment]
@acelina Thanks for your contribution.

Will be available on 1.26

**Pull request merged**